spec/unit/show-hide-content.spec.js

```diff
@@ -225,6 +273,10 @@ describe('show-hide-content', function () {
         }))
         expect(events && events.click).toContain(jasmine.objectContaining({
           namespace: 'ShowHideContent',
+          selector: 'input[type="radio"][name="test.single"]'
+        }))
+        expect(events && events.click).toContain(jasmine.objectContaining({
```

Comment:
Yeah, could be. As I was only looking to fix a specific bug with this change, I wasn’t looking to redesign the tests as part of the same change.